client/src/App.js

```diff
@@ -51,8 +57,9 @@ function App() {
   }
 
   async function checkAuth() {
-    // const GET_URL ='https://datafoundation.iiit.ac.in/api/detokn?token=' + tokenId
-    const GET_URL_DEV = 'http://10.4.25.20:3001/api/detokn?token=' + tokenId;
+    const GET_URL_DEV =
+      'https://datafoundation.iiit.ac.in/api/detokn?token=' + tokenId;
+    // const GET_URL_DEV = 'http://10.4.25.20:3001/api/detokn?token=' + tokenId;
```

Comment:
move this to constants

Reply:
Capital case is only used when it is a global constant

Reply:
resolved